contrib/platform/src/com/sun/jna/platform/win32/PsapiUtil.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2020 Torbj\u00F6rn Svensson, All Rights Reserved
```

Comment:
I just looked at other files and at least Advapi32Util had it like this. Should I change to the Unicode version instead?

Reply:
I'm not seeing what you're referring to in Advapi32Util.  My point is, a comment starting `/*` will only be seen in the source code, so if it's possible to include the correct character there (I'm pretty sure ö is allowed) that would be preferable.  For text to be rendered in HTML in javadocs (`/**`), use whatever formatting is required, as you have done.

Reply:
Sorry, I'm starting to confuse myself. It was actually in Psapi / PsapiTest. In any case, I'll switch to a Unicode 'ö' instead of the Unicode sequence in the comment.

Reply:
In anycase escaping is not necessary (or the build system needs to be adjusted). I only checked the javac calls, but these specify UTF-8 as encoding, so no problem using any unicode character in source.